definitions/control-v1.yaml

```diff
@@ -3801,6 +3807,251 @@ components:
+      - requestMode
+      - source
+      - target
+    pulsar_rule_patch:
+      type: object
+      additionalProperties: false
+      properties:
+        status:
+          type: string
+          description: The status of the rule. Rules can be enabled or disabled.
+          example: enabled
+        ruleType:
+          type: string
+          description: The type of rule. In this case Pulsar (using Firehose). See the <a href="https://ably.com/documentation/general/firehose">Ably documentation</a> for further information.
+          enum:
+          - pulsar
+        requestMode:
+          type: string
+          description: This is Single Request mode or Batch Request mode. Single Request mode sends each event separately to the endpoint specified by the rule. Batch Request mode rolls up multiple events into the same request. You can read more about the difference between single and batched events in the Ably <a href="https://ably.com/documentation/general/events#batching">documentation</a>.
+          example: single
+        source:
+          "$ref": "#/components/schemas/rule_source_patch"
+        target:
+          type: object
+          additionalProperties: false
+          properties:
+            routingKey:
+              type: string
+              description: The optional routing key (partition key) used to publish messages. Supports interpolation as described in the <a href="https://faqs.ably.com/what-is-the-format-of-the-routingkey-for-an-amqp-or-kinesis-reactor-rule">Ably FAQs</a>.
+              example: 'test-key'
+            topic:
+              type: string
+              description: 'A Pulsar topic. This is a named channel for transmission of messages between producers and consumers. The topic has the form: `{persistent|non-persistent}://tenant/namespace/topic`'
+              example: 'persistent://my-tenant/my-namespace/my-topic'
+            serviceUrl:
+              type: string
+              description: The URL of the Pulsar cluster.
+              example: 'pulsar://pulsar.us-west.example.com:6650/'
+            tlsTrustCerts:
+              type: array
+              description: All connections to a Pulsar endpoint require TLS. The `tlsTrustCerts` option allows you to configure different or additional trust anchors for those TLS connections. This enables server verification. You can specify an optional list of trusted CA certificates to use to verify the TLS certificate presented by the Pulsar cluster. Each certificate should be encoded in PEM format.
+              items:
+                type: string
+                example: "-----BEGIN CERTIFICATE-----
+MIIFiTCCA3GgAwIBAgIUYO1Lomxzj7VRawWwEFiQht9OLpUwDQYJKoZIhvcNAQEL
+BQAwTDELMAkGA1UEBhMCVVMxETAPBgNVBAgMCE1pY2hpZ2FuMQ8wDQYDVQQHDAZX
+...snip...
+TOfReTlUQzgpXRW5h3n2LVXbXQhPGcVitb88Cm2R8cxQwgB1VncM8yvmKhREo2tz
+7Y+sUx6eIl4dlNl9kVrH1TD3EwwtGsjUNlFSZhg=
+-----END CERTIFICATE-----"
+              nullable: true
+              example: 
```

Comment:
This `example` isn't required and causes the `tlsTrustCerts` to be null in the example request.

Reply:
Oops. Fixed.